packages/dds/matrix/src/permutationvector.ts

```diff
@@ -282,9 +282,9 @@ export class PermutationVector extends Client {
         // containing this position and use 'findReconnectionPosition' to adjust for the local ops that
         // have not yet been submitted.
 
-        // eslint-disable-next-line max-len
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/no-unnecessary-type-assertion
-        return this.findReconnectionPostition(containingSegment, localSeq) + containingOffset!;
+
```

Comment:
I hate this eslint bug... it leaves empty lines everywhere...

Reply:
Once I get the definitions and server version bumps done i'll get the eslint fix in, which will add back the rule to catch double-blank-lines